integration/nwo/commands/chaincodeserver.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package commands
+
+import "fmt"
+
+type ChaincodeServerStart struct {
+	Address    string
+	PackageID  string
+	ServerArgs []string
+}
+
+func (c ChaincodeServerStart) SessionName() string {
+	return fmt.Sprintf("%s-server", c.PackageID)
+}
+
+func (c ChaincodeServerStart) Args() []string {
+	return c.ServerArgs
+}
```

Comment:
This is not a command that is part of fabric; you should use a session specific to your test to launch your chaincode and shut it down.